pkg/rbac/configmap-adapter/adapter.go

```diff
@@ -50,6 +51,15 @@ func New(
 	}
 }
 
+// ConfigMap returns the configmap used for RBAC.
+func (a *Adapter) ConfigMap(ctx context.Context) (*corev1.ConfigMap, error) {
+	return a.kubeClient.GetConfigMap(
+		ctx,
+		a.namespacedName.Namespace,
+		a.namespacedName.Name,
+	)
+}
+
```

Comment:
You defined this function but I don't think you've used it anywhere...

Reply:
I did, it's in line 144 of the pkg/rbac/rbac.go file..

Reply:
My bad, sorry for that :see_no_evil: